src/fragments/lib-v1/interactions/js/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+
+<Callout warning>
+  The Interactions category utilizes the Authentication category behind the
+  scenes to authorize your app to send analytics events.
+</Callout>
+
+The `add` command automatically creates a backend configuration locally. To update your backend in the cloud, run:
+
+```bash
+amplify push
+```
+
+Upon successful execution of the push command, a configuration file called `aws-exports.js` will be copied to your configured source directory, for example `./src`.
+
+> If your Interactions resources were created with Amplify CLI version 1.6.4 and below, you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. [Read more](/cli/function/configure-options)
+
+## Manual setup
+### Lex V1 bot
+You can create Amazon Lex V1 chatbot in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex V1 Developer Guide](https://docs.aws.amazon.com/lex/latest/dg/getting-started.html).
+
+![Interactions](/images/interactions_lex_console_edit_bot.jpg)
+
+With manual setup, you need to provide your auth credentials and bot details to configure your app:
+
+```javascript
+import { Amplify } from 'aws-amplify';
+
+Amplify.configure({
+  Auth: {
+    identityPoolId: 'us-east-1:xxx-xxx-xxx-xxx-xxx',
+    region: 'us-east-1'
+  },
+  Interactions: {
+    bots: {
+      BookTrip: {
+        name: 'BookTrip',
+        alias: '$LATEST',
+        region: 'us-east-1'
+      }
+    }
+  }
+});
+```
+
+### Lex V2 bot
+You can create Amazon Lex V2 chatbot in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex V2 Developer Guide](https://docs.aws.amazon.com/lexv2/latest/dg/getting-started.html).
+
+![Interactions](/images/interactions_lex_v2_console_edit_bot.png)
+
+With manual setup, you need to provide your auth credentials and bot details to configure your app:
+
+```javascript
+import { Amplify } from 'aws-amplify';
+import { AWSLexV2Provider } from '@aws-amplify/interactions';
+
+Amplify.addPluggable(new AWSLexV2Provider());
+
+const interactionsConfig = {
+  Auth: {
+    identityPoolId: "<identityPoolId>",
+    region: "<region>"
+  },
+  Interactions: {
+    bots: {
+      // LexV2 Bot
+      <V2BotName>: {
+        name: "<V2BotName>",
+        aliasId: "<V2BotAliasId>",
+        botId: "<V2BotBotId>",
+        localeId: "<V2BotLocaleId>",
+        region: "<V2BotRegion>",
+        providerName: "AWSLexV2Provider",
+      },
+    }
+  }
+}
+
+Amplify.configure(interactionsConfig);
+```
+
+## Configure frontend
+
+import js0 from '/src/fragments/lib-v1/interactions/js/frontend.mdx';
+
+<Fragments fragments={{ js: js0 }} />
+
+import reactnative1 from '/src/fragments/lib-v1/react-native-polyfills.mdx';
+
+<Fragments fragments={{ 'react-native': reactnative1 }} />
+
+import reactnative2 from '/src/fragments/lib-v1/in-app-messaging/integrate-your-application/react-native/configure-amplify.mdx';
```

Comment:
This fragment 😅

Reply:
yeah, it's copied from the current doc on main branch. Maybe better to have a duplicated one in interactions module. How do you think?

Reply:
Nah let's just limit our changes to what's needed